design/scheduled-update-check.md

```diff
@@ -0,0 +1,64 @@
+# Scheduled Update Checks for Online Installs
+
+KOTS (Admin Console) does not automatically check for new updates. This proposal adds the 
+ability for KOTS (Admin Console) to check for updates every X period of time for online installations.
```

Comment:
This be a cron job style schedule.  The check should happen at specified time and not drift, which will happen if it's a time interval.

Reply:
:+1: this is a great suggestion.

Reply:
How would this work if the interval is configurable? Would it be better to use a cron job interval instead?

Reply:
There's a go package that converts cron spec to time.   I don't think an actual kubernetes CronJob should be used here.